src/promptflow/promptflow/_sdk/_orm/experiment.py

```diff
@@ -55,6 +56,8 @@ def dump(self) -> None:
                 if "UNIQUE constraint failed" not in str(e):
                     raise
                 raise ExperimentExistsError(f"Experiment name {self.name!r} already exists.")
+            except Exception as e:
+                raise UserErrorException(target=ErrorTarget.CONTROL_PLANE_SDK, message=str(e), error=e)
```

Comment:
Why is it a UserError here?

Reply:
I refer to `ExperimentExistsError`, which is `UserError`.

Reply:
According to offline discussions, we will further optimize the details for the next PR, and merge the current PR first.